src/ActionModal/components/ActionModal.vue

```diff
@@ -45,7 +45,7 @@
       <template v-if="!checkFeatureAvailable">
         <FeatureNotAvailable :feature="title" />
       </template>
-      <TmDataLoading v-else-if="!loaded" />
+      <TmDataLoading v-else-if="$apollo.loading && !$apollo.skipAll" />
```

Comment:
why?

Reply:
`loaded` was introducing too much unnecessary complexity and was responsible for the never loading modals bug. 
Also, if we set `skipAll` to true, it doesn't make sense to wait for data. I think this is the most logical.

Reply:
loaded != $apollo.loading. loading changes every loading call loaded changes only after the first load

Reply:
ping

Reply:
I changed to `$apollo.queries.<query>.loading`, but it is probably the same.

I will just copy from Undelegations table. There I remember it had the proper logic

Reply:
using "loading" will show the loader on every apollo load of balances or overview

Reply:
Yes, I was thinking. Thank you for confirming. I just fixed the logic of it all